src/frontend/components/UI/DLCList/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { DLCInfo } from 'common/types/legendary'
+import DLC from './DLC'
+import './index.scss'
+import { GameInfo, Runner } from 'common/types'
+
+type Props = {
+  dlcs: DLCInfo[]
+  runner: Runner
+  mainAppInfo: GameInfo
+  onClose: () => void
+}
+
+const DlcList = ({ dlcs, runner, mainAppInfo, onClose }: Props) => {
+  const { t } = useTranslation()
+
+  return (
+    <div className="dlcList">
+      <div className="dlcHeader">
+        <span className="title">{t('dlc.title', 'Title')}</span>
+        <span className="size">{t('dlc.size', 'Size')}</span>
+        <span className="actions">{t('dlc.actions', 'Actions')}</span>
+      </div>
+      {dlcs.length > 0 &&
```

Comment:
So, at first it was only accessible if we have DLCs, but the problem is that there is a delay to check for the DLCs since we need to wait for the `legendary info` command to finish. So then the menu was hidden at first and then was appearing. to avoid this behavior I decided to show it for all games and check for the owned DLCs afterwards.